### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Blob Container Table_Screen Reader: NVDA/Narrator is not announcing the sorting information present for the column headers.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/AzureStorageExplorer#7585]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1 (96)

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

All

### OS Version

Windows 11 Enterprise Insider Preview (23H2)

### Bug Description
GitHub Tags:
#A11yMAS;#A11yTCS;#AILimited;#Win11;#DesktopApp;#A11ySev3;#BM_AzureToolsStorageExplorer_Win32_Dec2023;#Azure Tools Storage Explorer;#FTP;#ScreenReader;#WCAG1.3.1;#NVDA;#Element:Table;#STP;#Narrator;#Infoand Relationships;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)
Screen Reader: Narrator, NVDA - version 2023.1

### Steps to Reproduce

1. Turn on Narrator/NVDA.
2. Launch Storage Explorer
3. Tab till "Open Connect dialog" pane and Sign in to Azure.
4. Tab till "Manage Accounts" pane and Select the subscriptions you will use. Once selected, hit enter on "Apply" button.
5. Tab till "Blob Containers" of the selected subscription, press context menu and hit enter on "Create Blob container" to Create a new Blob container.
6. Once completed, tab till "Upload" button and Upload a blob/file by using "Upload Files" control.
7. Tab till Table present in right side of the storage explorer.
8. Verify that NVDA/Narrator is announcing the sorting information present for the column headers or not.

### Actual Experience

On performing sorted functionality in column headers screen reader is not announce the sorting information like "sorted ascending" or "sorted descending".

### Expected Experience

On performing sorted functionality in column headers screen reader should announce the sorting information like "sorted ascending" or "sorted descending". Also when we navigate to the particular column header screen reader should announce the current sorting state.

### User Impact:
Screen Reader users will miss the sorting functionality if screen reader is not announcing the sorting information.

### Attachment:
![A11y_Azure Tools Storage Explorer_Blob Container Table_Screen Reader_Narratorr](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/5d651920-51a9-430d-a667-28a58b94d0b3)

**Comment by Mana-swi** [comment]
#a11ydb-sort-not-announced;
